src/libs/TransactionUtils.ts

```diff
@@ -520,6 +521,13 @@ function getRecentTransactions(transactions: Record<string, string>, size = 2):
         .slice(0, size);
 }
 
+/**
+ * Checks if any violations for the provided transaction are of type 'violation'
+ */
+function hasViolation(transactionID: string, transactionViolations: OnyxCollection<TransactionViolation[]> | undefined): boolean {
```

Comment:
Can we remove `undefined` as a valid param value for `transactionViolations`?

Reply:
done